awesomeConfig.cmake

```diff
@@ -67,6 +67,12 @@ a_find_program(CONVERT_EXECUTABLE convert TRUE)
 include(FindPkgConfig)
 # lua
 include(FindLua)
+if (NOT LUA_FOUND)
+    message(FATAL_ERROR
+        "Could not find Lua. See the error above.\n"
+        "You might want to hint it using the LUA_DIR environment variable, "
+        "or set the LUA_INCLUDE_DIR / LUA_LIBRARY CMake variables.")
+endif()
```

Comment:
That would work:
```
-- Found PkgConfig: /usr/bin/pkg-config (found version "1.3.7") 
CMake Error at /usr/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:137 (message):
  Could NOT find Lua (missing: LUA_LIBRARIES) (found version "5.2.4")
Call Stack (most recent call first):
  /usr/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:377 (_FPHSA_FAILURE_MESSAGE)
  /usr/share/cmake/Modules/FindLua.cmake:193 (FIND_PACKAGE_HANDLE_STANDARD_ARGS)
  awesomeConfig.cmake:69 (find_package)
  CMakeLists.txt:32 (include)


-- Configuring incomplete, errors occurred!
```

However, I think it is useful to have the extra message from the custom `if` block:
```
-- Found PkgConfig: /usr/bin/pkg-config (found version "1.3.7") 
-- Could NOT find Lua (missing:  LUA_LIBRARIES) (found version "5.2.4")
CMake Error at awesomeConfig.cmake:71 (message):
  Could not find Lua.  See the error above.

  You might want to hint it using the LUA_DIR environment variable, or set
  the LUA_INCLUDE_DIR / LUA_LIBRARY CMake variables.
Call Stack (most recent call first):
  CMakeLists.txt:32 (include)


-- Configuring incomplete, errors occurred!
```

What do you think?